docs/validate.md

```diff
@@ -0,0 +1,9 @@
+This document explains the changes made in the identifier validation logic between the containerd identifier implementation and the Docker compatible implementation `ValidateDockerCompat`.
```

Comment:
This document should be moved to godoc of ValidateDockerCompat

Reply:
i am assuming the suggestion is to rename the file to: ValidateDockerCompat

Reply:
ah ok will do